src/main/resources/stock_trades_schema.avro

```diff
@@ -0,0 +1,64 @@
+{
+  "namespace": "ksql",
+  "name": "StockTrade",
+  "doc": "Defines a hypothetical stock trade using some known simulated stock symbols.",
+  "type": "record",
+  "fields": [
+    {
+      "name": "side",
+      "doc": "A simulated trade side (buy or sell or short)",
+      "type": {
+        "type": "string",
+        "arg.properties": {
+          "options": [ "BUY", "SELL", "SHORT" ]
+        }
+      }
+    },
+    {
+      "name": "quantity",
+      "doc": "A simualted random quantity of the trade",
+      "type": {
+        "type": "int",
+        "arg.properties": {
+          "range": {
+            "min": 1,
+            "max": 5000
+          }
+        }
+      }
+    },
+    {
+      "name": "symbol",
+      "doc": "Simulated stock symbols",
+      "type": {
+        "type": "string",
+        "arg.properties": {
+          "options": [ "ZJZZT","ZVZZT","ZWZZT" ]
+        }
+      }
+    },
+    {
+      "name": "price",
+      "doc": "A simulated random trade price in pennies",
+      "type": {
+        "type": "int",
+        "arg.properties": {
+          "range": {
+            "min": 5,
+            "max": 1000
+          }
+        }
+      }
+    },
+    {
+      "name": "account",
+      "doc": "Simualted accounts assigned to the trade",
```

Comment:
"simualted" --> "simulated"
All of this is simulated, so it is redundant to say it everywhere